tests/test_data_integrity.py

```diff
@@ -1,18 +1,19 @@
 from operator import itemgetter
 import unittest
 
-from whotracksme.data import load_tracker_db, load_apps, load_companies
+from whotracksme.data import load_tracker_db, DataSource
 
 
 class TestDataIntegrity(unittest.TestCase):
 
     def setUp(self):
         self.conn = load_tracker_db()
+        self.ds = DataSource()
 
     def test_all_trackers_have_db_entry(self):
-        apps = load_apps()
+        trackers = self.ds.trackers
         cur = self.conn.cursor()
-        app_ids = sorted(apps.keys())
+        app_ids = trackers.get_snapshot().tracker
```

Comment:
Given the table itself is the tracker table, I think it would be more intuitive to have the column named `id`, so `trackers.get_snapshot().id`